templates/openstack/consulting.html

```diff
@@ -8,68 +8,218 @@
-        <p><a href="{{ ASSET_SERVER_URL }}8cf6fe4f-Foundation-Cloud-Build-Openstack-data-sheet.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read the datasheet', 'eventLabel' : 'Read the datasheet - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Read the datasheet</a>&#8195;<a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Talk to us', 'eventLabel' : 'Talk to us - Top CTA', 'eventValue' : undefined });" class="p-button--neutral">Talk to us</a></p>
+        <h1>OpenStack consulting, design and delivery</h1>
+        <h2 class="p-heading--three">Carrier grade, banking security, government scale. Serious private clouds are our business.</h2>
+        <p>Hundreds of successful OpenStack clouds underpin our consulting, training, architecture design, and hardware procurement advice - to help you evaluate and deliver OpenStack.</p>
+        <p>
+          <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Engage Canonical exports', 'eventLabel' : 'Engage Canonical exports - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Engage Canonical experts</a>
+          &#8195;
+          <a href="/openstack/canonical-openstack-consulting.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Read our consulting datasheet', 'eventLabel' : 'Read our consulting datasheet - Top CTA', 'eventValue' : undefined });" class="p-button--neutral">Read our consulting datasheet</a>
+        </p>
       </div>
     </div>
   </section>
 
   <section class="p-strip is-bordered">
-    <div class="row p-divider u-equal-height">
+    <div class="row">
+      <h2>Canonical will design and deploy OpenStack on your premises</h2>
+    </div>
+    <div class="row">
+      <div class="u-equal-height">
+        <div class="col-8">
+          <p>Building and running an OpenStack cloud requires specialised expertise and low-level Linux OS knowledge. Canonical brings the full stack &mdash; from kernel to containers to applications - along with operating experience from hundreds of OpenStack deployments around the world to get you started efficiently.</p>
+          <p>Once you are certain OpenStack is the right choice we can deliver your cloud in two weeks. Only then do you need to commit the devops talent required to run your cloud, or <a href="/openstack/managed">outsource that to Canonical</a> as well.</p>
+        </div>
+        <div class="col-4 u-hide--small u-vertically-center">
```

Comment:
personally, I would add 'prefix-1' and make the illustration smaller

![image](https://user-images.githubusercontent.com/441217/41773536-47366bde-7614-11e8-8d85-d8970168919a.png)

Reply:
I set a max-width on it as @anasereijo requested it so this shouldn't be necessary now